altar/src/render/brisk.rs

```diff
@@ -280,22 +279,68 @@ impl Drop for Spritesheets {
 
 impl Spritesheets {
     /// Creates a new spritesheet given a path and a [`Vec<F32Box2>`], representing the
-    /// UVs. Returns the [`SpritesheetId`] for the new spritesheet (in the event that the
-    /// specified path was already in present, the old SpritesheetId will be returned).
-    pub fn new_sheet(&mut self, path: &str, uvs: Vec<F32Box2>) -> SpritesheetId {
+    /// frames. Returns the [`SpritesheetId`] for the new spritesheet (in the event that the
+    /// specified path was already present, the old SpritesheetId will be returned).
+    pub fn new_sheet(&mut self, path: &str, frames: Vec<Frame>) -> SpritesheetId {
+        if let Some(ssid) = self.path_map.get_mut(path) {
+            self.uncached.insert(*ssid);
+            *ssid
+        } else {
+            let full: StaticRc<str, 2, 2> = path.into();
+            let (half_1, half_2) = Full::split::<1, 1>(full);
+            let id = self.ss_arena.insert(Spritesheet {
+                path: half_1,
+                frames,
+            });
+            let ssid = id.into();
+            self.path_map.insert(half_2, ssid);
+            self.uncached.insert(ssid);
+            ssid
+        }
+    }
+
+    #[cfg(feature = "aseprite")]
+    /// Creates a new spritesheet given an aseprite [`aseprite::SpritesheetData`]. Aseprite
+    /// frames are indexed using the `frame_id` field defined in the [`Sprite`] type.
+    /// Returns the [`SpritesheetId`] for the new spritesheet (in the event that the
+    /// specified path was already present, the old SpritesheetId will be returned).
+    pub fn new_from_ase_sheet(&mut self, ase_sheet: &aseprite::SpritesheetData) -> SpritesheetId {
```

Comment:
I'd prefer not to shorten things unless they're really long, so can we call this `new_from_aseprite_sheet`?